src/components/Menu.test.tsx

```diff
@@ -57,4 +57,11 @@ describe('Menu', () => {
     await user.click(getByText('Delete'));
     expect(mockOnClick).toHaveBeenCalledWith(Action.delete);
   });
+
+  it('renders disabled state when readOnly is true', async () => {
```

Comment:
```suggestion
  it('renders disabled state when readOnly is true', () => {
```

Reply:
Since we're not using await here we don't need to specify `async` for this function :)